composer.json

```diff
@@ -35,7 +37,7 @@
         "liip/functional-test-bundle": "~1.0",
         "sensio/generator-bundle": "~2.3",
         "sensio/distribution-bundle": "~3.0,>=3.0.12",
-        "sensio/framework-extra-bundle": "~3.0",
+        "sensio/framework-extra-bundle": "~3.0,>=3.0.2",
```

Comment:
Yes, but it was included in a beta release less than one month ago, so we have to wait a couple more weeks before we can start using it.